cesium/options.go

```diff
@@ -51,6 +53,12 @@ func WithFS(fs xfs.FS) Option {
 	}
 }
 
+func WithGC(config *GCConfig) Option {
```

Comment:
Why do you need to pass this as a pointer? Also, where do you merge in the default options that you defined?